go/private/actions/compilepkg.bzl

```diff
@@ -117,6 +121,10 @@ def emit_compilepkg(
         outputs.append(out_cgo_export_h)
     if testfilter:
         args.add("-testfilter", testfilter)
+    minor = minor_version(go.sdk.version)
+    if minor != None and minor >= 20:
+        # Turn off coverageredesign GOEXPERIMENT
+        args.add("-nocoverageredesign")
```

Comment:
Do you expect compatibility with older SDKs if we were to always set this flag?

Reply:
Do you mean if we always set GOEXPERMENT=nocoverageredesign on older SDKs? That wouldn't be compatible because the Go SDK tools will produce errors if GOEXPERIMENT contains experiment names that are unknown to older versions of the SDK. So setting GOEXPERIMENT=nocoverageredesign on `go install` with Go 1.18, for example, will fail because the coverageredesign experiment was unknown to that version of Go.